src/docs/stan-reference/functions.tex

```diff
@@ -3119,6 +3119,15 @@ \subsubsection{QR Decomposition}\label{QR-decomposition}
 
 \begin{description}
 %
+\fitem{matrix}{qr\_thin\_Q}{matrix \farg{A}}{
+The orthogonal matrix in the thin QR decomposition of \farg{A}, which
+implies that the resulting matrix is the same dimensions as as \farg{A}}
+%
+\fitem{matrix}{qr\_thin\_R}{matrix \farg{A}}{
+  The upper triangular matrix in the thin QR decomposition of \farg{A},
+  which implies that the resulting matrix has the same dimensions as
```

Comment:
"the resulting matrix is square with the same number of columns as A."

Just to emphasize it is square so both dimensions are the same. This is the only necessary change.